src/Components/Bulletins/Header/BulletinHeader.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Header: React.FC = () => {
+  return (
+    <div>
+      <h1 className="ui dividing centered header">BULLETINS</h1>
+      <hr />
+    </div>
```

Comment:
Looks like our styling adds a line under the header. Can we remove the `<hr />` tag and the surrounding div?